### PR TITLE
HPCC-32398 update references to system requirements

### PR DESCRIPTION
**Pull request by g-pan**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [x] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

https://github.com/g-pan/github-action-dev-build/actions/runs/10394628523

### CODE REVIEW

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
The PT_BR version of Hardware.xml also needs to be removed.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
@g-pan Let me know when build issue is resolved.

**Review by JamesDeFabia — APPROVED** [review]
Looks good

**Comment by JamesDeFabia** [comment]
Make sure @vfumene knows about https://hpccsystems.atlassian.net/browse/HPCC-32445

File: docs/PT_BR/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 1]
```diff
@@ -442,7 +442,7 @@
     for implementar ou dimensionar seu HPCC System. As sugestões devem ser
     levadas em conta de acordo com as suas necessidades empresariais
     específicas.</para>
-
+<!-- COMMENTING FOR TEST - THIS WHOLE SECTION SHOULD BE REMOVED.
```
Comment (ghalliday) [inline docs/PT_BR/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:445]: Are you going to remove this section?

**Review by ghalliday — COMMENTED** [review]
@g-pan will need squashing, but it also looks like a section of the file should be deleted.

**Comment by g-pan** [comment]
something is not right here, squashed yet git is not reflecting that. 
Closing this request to reissue cleanly.

**Pull request closed without merging**